tests/testthat/test_parameters.R

```diff
@@ -19,7 +19,10 @@ test_that('param ranges', {
   expect_equal(num_comp(1:2)$range, list(lower = 1L, upper = 2L))
   expect_equal(cost(-2:-1)$range, list(lower = -2, upper = -1))
   expect_equal(scale_factor(-2:-1)$range, list(lower = -2, upper = -1))
-  expect_equal(dials::margin(c(.1, .15))$range, list(lower = .1, upper = .15))
+  expect_warning(
```

Comment:
Can you please pull out this test and move it to `test_deprecated.R` ?

Reply:
Done. Also `dials::` from previous version was removed